code/__HELPERS/roundend.dm

```diff
@@ -711,3 +714,31 @@
 		Q.Execute()
 		qdel(Q)
 
+/datum/controller/subsystem/ticker/proc/antag_rounds()
+	var/list/antagckeys = list()
+	var/list/nonantagckeys = list()
+	var/numberofrounds = 1
+	
+	for(var/datum/mind/allminds in GLOB.startingminds)
+		var/list/antagdatums = allminds.antag_datums
+		for(var/datum/antagonist/antagdatum in antagdatums)
+			if(antagdatums)
+				if(!antagdatum.owner)
+					continue
+				if(!antagdatum.restrictmultiplerounds)
+					continue
+				antagckeys += antagdatum.owner.key
+			nonantagckeys += antagdatum.owner.key
+	
+
+	var/datum/DBQuery/Q = SSdbcore.New("UPDATE [format_table_name("player")] SET `antag_rounds` = GREATEST('antag_rounds` -1, 0) WHERE `key` IN ([antagckeys])")
```

Comment:
This needs to be formatted as `('item', 'item2', ....)` will byond format the string this way? You might need to manually loop through the ckey list and generate a string